### PR TITLE
use gripper v3 for left arm in stow main program

### PR DESCRIPTION
**Pull request by knorth55**

### CODE REVIEW

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l [hunk 1]
```diff
@@ -812,15 +811,8 @@
...
-        (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a sign_y: ~a" arm sign sign_y)
-        (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a" arm sign))
-      (if (eq arm :larm)
-        (progn
-          ;; start the vacuum gripper
-          (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a start vacuum gripper" arm)
-          (send self :start-grasp arm)
-          (unix::sleep 1)) nil)
+      (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a sign_y: ~a" arm sign sign_y)
+      (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a" arm sign)
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: This is not needed

File: jsk_2016_01_baxter_apc/euslisp/main-stow.l [hunk 2]
```diff
@@ -97,37 +97,38 @@
...
+              (ros::ros-info "[main] ~a, ~a" arm state)
+              (ros::ros-info "[main] ~a, place object in bin ~a" arm target-bin)
+              (send *ri* :move-arm-body->bin arm target-bin)
+              (send *ri* :wait-interpolation)
+              (send *ri* :gripper-servo-off arm)
+              (setq avs-picked->return-bin
                     (mapcar
                       #'(lambda (x)
                           (send *ri* :ik->bin-entrance arm target-bin
                                 :offset (float-vector x 0 -20)))
```
Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/main-stow.l]: This is ok for now but please update this like #1670 in another PR.

**Comment by knorth55** [comment]
@pazeshun sorry, i made wrong commmit, and fixed

File: jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l (hunk 1 above)
Comment (knorth55) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: why you do not need this info?

Comment (pazeshun) [inline jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l]: `(ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a" arm sign)`
This is for old left arm.
`(ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a sign_y: ~a" arm sign sign_y)` is sufficient.

**Comment by knorth55** [comment]
all fixed

**Comment by pazeshun** [comment]
👍

**Comment by pazeshun** [comment]
Please test return_object. If you think the movement is strange, please tell me

**Pull request merged**